math/octave-devel/Portfile

```diff
@@ -171,5 +171,97 @@ configure.args-append       --disable-silent-rules                  \
                             --disable-java                          \
                             --with-qt=4
 
-                            
+variant app description "build application bundle to launch octave-devel" {
+    depends_build-append port:librsvg
+
+    if {[vercmp ${xcodeversion} 4.5] < 0} {
+        # need a way to generate icns file for XCode prior to 4.5
+        # see #51487
+        depends_build-append port:libicns
+    }
+
```

Comment:
these last two deps are also up in the build_deps section, along with some env-appends for the executables. I guess we should move those down here, with the env-appends as well...